WordPress/Classes/ViewRelated/Gutenberg/GutenbergMediaInserterHelper.swift

```diff
@@ -153,7 +153,21 @@ class GutenbergMediaInserterHelper: NSObject {
             guard let urlString = media.remoteURL, let url = URL(string: urlString), let mediaServerID = media.mediaID?.int32Value else {
                 break
             }
-            gutenberg.mediaUploadUpdate(id: mediaUploadID, state: .succeeded, progress: 1, url: url, serverID: mediaServerID)
+            if media.mediaType == .image {
```

Comment:
Maybe a switch will be better, just in case in the future we add some extra cases.

Reply:
Done